azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+trigger:
```

Comment:
Did you hook this up to a pipeline anywhere and test it?  How do you expect me to hook it up to CI?

Reply:
Setup a free Azure DevOps pipeline? :)

I haven't tested it but I'm mostly sure it works ;) It's based on many others I've done.